copr-gpg-download.py

```diff
@@ -1,51 +1,63 @@
 #!/usr/bin/python3
 
+from copr.v3 import Client
+from copr.v3.pagination import next_page
 from copr import create_client2_from_params
+
 import requests
 import argparse
 from argparse import RawTextHelpFormatter
 import os.path
 
-def get_gpg(project):
-    url = be_url_tmpl.format(**{'username': project.owner, 'projectname': project.name})
+
+def get_gpg(url):
+
     r = requests.get(url)
     return r.text
 
-def gpg_out(isolate_file, project):
+
+def gpg_out(isolate_file, file_name, url):
     if isolate_file:
-        file_name = "copr-{0}-{1}.gpg".format(project.owner, project.name)
         if not os.path.isfile(file_name):
-            f = open(file_name, 'w')
-            f.write(get_gpg(project))
-            f.close()
+            with open(file_name, "w") as f:
```

Comment:
I tried to run:  `pushd keys/copr && ../../copr-gpg-download.py --isolate-files && popd` and I got:
```
Traceback (most recent call last):
  File "../../copr-gpg-download.py", line 89, in <module>
    main()
  File "../../copr-gpg-download.py", line 46, in main
    gpg_out(args.isolate_files, file_name, url)
  File "../../copr-gpg-download.py", line 22, in gpg_out
    with open(file_name, "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'keys/copr/copr-@mock-mock.gpg'
```